linux_os/guide/system/bootloader-grub2/grub2_mitigation_argument/rule.yml

```diff
@@ -32,17 +32,32 @@ references:
 
 platform: grub2
 
-ocil_clause: 'mitigation is not set to on'
+ocil_clause: 'mitigations is set to off'
 
 ocil: |-
-    {{{ ocil_grub2_argument("mitigation=on") | indent(4) }}}
+    Make sure that the kernel is not disabling mitigations with the following
+    commands.
+    <pre>$ sudo grubby --default-kernel</pre>
```

Comment:
If you willing to use the `ALL` verses the specific kernel these lines can be moved.

I understand that this isn't following STIG to the letter, but this will ensure that all install kernels are updated. This also more consistent internally within the project.